You may want to mention the Gradle versions this plugin is compatible with. Since NullAway had too many false positives, I tried this but it is incompatible with Gradle 4.4

```
* What went wrong:
A problem was found with the configuration of task ':caffeine:inferPrepare'.
> No value has been specified for property 'bootClasspath'.
...
Caused by: org.gradle.api.InvalidUserDataException: No value has been specified for property 'bootClasspath'.
        at org.gradle.api.internal.tasks.TaskValidationContext$Severity$2.report(TaskValidationContext.java:51)
```

--- a/README.md
+++ b/README.md
@@ -4,9 +4,9 @@
 
 This Gradle plug-in creates tasks to run [Infer](http://fbinfer.com) on Android and Java projects.
 
-## DEPRECATED
+## Deprecated
 
-This plug-in isn't used or maintained anymore. We strongly recomend migrating [NullAway](http://github.com/uber/nullaay) which is much more reliabile, significantly faster, and actively maintained.
+This plug-in isn't used or maintained anymore. For null checking, we recomend migrating [NullAway](http://github.com/uber/nullaay). If you're still using Infer for additional checks, we recomend investigating other approaches as this plug-in isn't actively being worked on.
 
 ## Integration
 

--- a/README.md
+++ b/README.md
@@ -6,7 +6,7 @@ This Gradle plug-in creates tasks to run [Infer](http://fbinfer.com) on Android 
 
 ## Deprecated
 
-This plug-in isn't used or maintained anymore. For null checking, we recomend migrating [NullAway](http://github.com/uber/nullaay). If you're still using Infer for additional checks, we recomend investigating other approaches as this plug-in isn't actively being worked on.
+This plug-in isn't used or maintained by the original authors anymore. For null checking, we recommend migrating to [NullAway](http://github.com/uber/NullAway). If you're still using Infer for additional checks, we recomend investigating other approaches as this plug-in isn't actively being worked on.
 
 ## Integration
 

